tests/TestRunner/app/Inheritance/InheritanceTests.js

```diff
@@ -1846,39 +1846,54 @@ describe(module.id, function () {
         expect(NSString.stringWithCString(encoding).toString()).toBe("v@:");
     });
 
-    it("should project Symbol.iterable as NSFastEnumeration", function () {
-        var start = 1;
-        var end = 10;
-        var lastStep = 0;
-        var IterableClass = NSObject.extend({
-            [Symbol.iterator]() {
-                return {
-                    step: start,
-
-                    next() {
-                        if (this.step <= end) {
-                            return { value: this.step++, done: false };
-                        } else {
-                            return { done: true };
-                        }
-                    },
-
-                    return() {
-                        lastStep = this.step;
-                        return {};
-                    }
-                }
+    it("should project Symbol.iterable as NSFastEnumeration", function() {
+      var start = 1;
+      var end = 10;
+      var lastStep = 0;
+      var IterableClass = NSObject.extend({
+        [Symbol.iterator]() {
+          return {
+            step: start,
+
+                next() {
+              if (this.step <= end) {
+                return {value : this.step++, done : false};
+              } else {
+                return {done : true};
+              }
             }
-        });
+            ,
 
-        var expected = "12345678910";
-        TNSIterableConsumer.consumeIterable(IterableClass.alloc().init());
-        var actual = TNSGetOutput();
+                return () {
+              lastStep = this.step;
+              return {};
+            }
+          }
+        }
+      });
 
-        expect(IterableClass.conformsToProtocol(NSFastEnumeration)).toBe(true);
-        expect(actual).toEqual(expected);
-        expect(lastStep).toEqual(end + 1);
+      var expected = "12345678910";
+      TNSIterableConsumer.consumeIterable(IterableClass.alloc().init());
+      var actual = TNSGetOutput();
 
-        TNSClearOutput();
-    })
+      expect(IterableClass.conformsToProtocol(NSFastEnumeration)).toBe(true);
+      expect(actual).toEqual(expected);
+      expect(lastStep).toEqual(end + 1);
+
+      TNSClearOutput();
+    });
+
+    it("Mehtod and property with the same name", function() {
+      var JSObject =
+          NSObject.extend({get conflict() { return true; }},
+                          {protocols : [ TNSPropertyMethodConflictProtocol ]});
+
+      var derived = JSObject.new();
+      var result;
+      expect(function() {
+        result = TNSTestNativeCallbacks.protocolWithNameConflict(derived);
+      }).not.toThrow();
```

Comment:
I think that this expectation is redundant - all calls shouldn't throw an error and the test will fail if such happens.